index.html

```diff
@@ -1,12 +1,169 @@
+              <a class="navigation-link" href="about.html">О Седоне</a>
+            </li>
+            <li class="navigation-item">
+              <a class="navigation-link" href="catalog.html">Гостиницы</a>
+            </li>
+          </ul>
+          <ul class="user-navigation-list">
+            <li class="user-navigation-item">
+              <a class="user-navigation-link" href="#" >
+                <span class="visually-hidden">Лупа для поиска (картинка)</span>
+              </a>
+            </li>
+            <li class="navigation-item">
+              <a class="navigation-link" href="#">
+                <span class="visually-hidden">
+                  избранное(картинка)
+                </span>
+              </a>
+            </li>
+            <li class="navigation-item">
+              <a class="navigation-link" href="#">
+                <span class="visually-hidden">
+                  ХОЧУ СЮДА(кнопка)
+                </span>
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+      <main class="main-index">
+          <h1 class="visually-hidden">
+            SEDONA
+          </h1>
+          <img class="picture-sedona" src="#" alt="Слоган Седоны">
+          <p>"Седона — небольшой городок в Аризоне, заслуживающий большего!"</p>
+          <p>Рассмотрим причины, по которым Седона круче, чем Гранд-Каньон!</p>
+          <section class="advantages">
+              <h2 class="visually-hidden">Преимущества</h2>
+          <ul class="advantages-list">
+            <li class="advantages-item">
+              <h3 class="advantages-title">Настоящий городок</h3>
+              <p class="advantages-description">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
+              <img class="logo" src="#" alt="картинка города">
+            </li>
+            <li class="advantages-item">
+              <h3 class="advantages-title">Небольшая площаль</h3>
+              <p class="advantages-description">Все достопримечательности находятся очень близко</p>
+            </li>
+            <li class="advantages-item">
+              <h3 class="advantages-title">Красивая дорога</h3>
+              <p class="advantages-description">Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
+            </li>
+            <li class="advantages-item">
+              <h3 class="advantages-title">Мало туристов</h3>
+              <p class="advantages-description">Большинство едет в Гранд Каньон и толпится там</p>
+            </li>
+            <li class="advantages-item">
+              <h3 class="advantages-title">Там есть мост дьявола</h3>
+              <p class="advantages-description">Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+              <img class="logo" src="#" alt="картинка Моста Дьявола">
+            </li>
+          </ul>
+          </section>
+          <section>
+            <h3 class="services-title visually-hidden">Предложения</h3>
+          </section>
```

Comment:
```suggestion
          <section>
            <h3 class="services-title visually-hidden">Предложения</h3>
```
Там дальше идут предложения, которые тоже относятся к этой секции